### ISSUE TITLE
Allow 'Based on' to be customizable changing the entire text.

### ISSUE THREAD

**Issue by dmundra** [CivicActions/openacr-editor#10]
Should be stored in YAML. Example:

```
<p>Format: <b>VPAT 2.4Rev WCAG</b></p>
```
